index.css

```diff
@@ -0,0 +1,586 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap');
+.container{
+    margin: 0px;
+    font-family: 'poppins';
```

Comment:
if you don't have multiple fonts, consider using the universal selector to apply a uniform font throughout.

Reply:
I've already looked into that method, but Koni suggested that using universal selectors might lead to performance issues. That's why I chose to go with this option.

Reply:
ok , let it be